lib/ProductOpener/Ingredients.pm

```diff
@@ -3966,8 +3968,8 @@ my %phrases_after_ingredients_list = (
 		'once opened[,]? (consume|keep|refrigerate|store|use)',
 		'(Storage( instructions)?[: ]+)?Store in a cool[,]? dry place',
 		'(dist(\.)?|distributed|sold)(\&|and|sold| )* (by|exclusively)',
-		#'Best before',
 		#'See bottom of tin',
+		'spices and or vegetable powder as last ingredient',
```

Comment:
What's the product with this string?

Reply:
2252559300126
See my comment in the main message of the PR. There is a single product with that text after the ingredients list. 
Is it an overkill to have it in the code? Do you prefer that I remove it?

Reply:
I think we should remove it, in fact in this product there's also an "and SPINACH as second ingredient when that flavor -" that we won't recognize either.